.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: python
 dist: bionic
 python:
-  - "2.7"
-  - "3.5"
   - "3.6"
   - "3.7"
   - "3.8"
```

Comment:
can we add 3.9 (maybe also 3.10)?

Reply:
running tests on 3.10 will be really nice, but it shouldn't block from merging, so let's remember about adding `ignore_outcome = true` for py310 env in `tox.ini`

Reply:
still, can you add python 3.9?
This PR it's not about applying black to a file anymore, so we could try to add python 3.10 to the CI matrix as well.
Please also change the title of this PR to reflect what it actually does.